Assets/Scripts/Interaction Scripts/ControllerState.cs

```diff
@@ -0,0 +1,195 @@
+﻿namespace ISAACS {
+
+    using System.Collections;
+    using System.Collections.Generic;
+	using VRTK;
+    using UnityEngine;
+
+	public class ControllerState : MonoBehaviour {
+
+		// Remember to set these controllers to actual GameObjects through the Unity GUI.
+		public GameObject LeftController;
+		public GameObject RightController;
+		public GameObject controller_left;
+		public GameObject controller_right;
+
+		// True indicates that the button is being pressed, False that it's released.
+		private bool LeftIsGrabbing; // Whether the left hand is grabbing another object.
+		private bool LeftIndex;
+		private bool LeftMiddle;
+		private bool LeftX;
+		private bool LeftY;
+		private bool LeftThumb;
+		private Vector2 LeftThumbDelta; // The distance by which the left stick has been moved in the X-axis and Y-axis.
```

Comment:
Minor nit pick but these description comments can be added to the Getter methods below as well.

Using C# `///` comments will showup in IDE help displays.